wgpu-core/Cargo.toml

```diff
@@ -21,6 +21,7 @@ replay = ["serde", "wgt/replay", "arrayvec/serde", "naga/deserialize"]
 # Enable serializable compute/render passes, and bundle encoders.
 serial-pass = ["serde", "wgt/serde", "arrayvec/serde"]
 id32 = []
+vulkan = ["hal/vulkan"]
```

Comment:
Let's call it vulkan-portability to avoid confusion (i.e. you don't need the feature for running actual Vulkan)

Reply:
already changed feature name to vulkan-portability